src/BluetoothListPage.js

```diff
@@ -0,0 +1,65 @@
+
+import React from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  useColorScheme,
+  View,
+  FlatList
+} from 'react-native';
+
+import {H1,H2,H3,Body,BodySecondary} from './Components/typography'
+import {Header, CellContainer, CellTitleContainer, VBox, VSeparator, ListSeparator, ListCell} from './Components/cell'
+import {BTList, BluetoothEnableButton} from './Components/BTList'
```

Comment:
```suggestion
import {H1,H2,H3,Body,BodySecondary} from './components/typography'
import {Header, CellContainer, CellTitleContainer, VBox, VSeparator, ListSeparator, ListCell} from './components/cell'
import {BTList, BluetoothEnableButton} from './components/BTList'
```
sem mudar isso não funcionava no meu celular